### ISSUE TITLE
There is no functionality provided to adjust columns width which is achievable using single pointer.:A11y_.NetAspireDashboard_Resources_Grid_DraggingMovement

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3364]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #A11yWCAG2.2; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #DraggingMovement; #WCAG2.5.7;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:
1. Hit the URL: http://localhost:15888/
2. TAB to "Resources" tab item and hit select the tab by pressing Enter key.
3. Navigate on the columns and check if single pointer functionality is provided to adjust the width of column.

### Actual Results:
There is no functionality provided to adjust columns width which is achievable using single pointer.

**Note: Same issue reproes on 'Structured Logs' page -> table and Traces page->table.**

### Expected Results:
There should be functionality provided to adjust columns width which is achievable using single pointer.

### User Impact:
Functionality provided only with dragging will create difficulty for users using input devices like a trackball, head pointer, eye-gaze system to perform the operation.

### Attachment:
[3364_A11y_.NetAspireDashboard_Resources_Grid_DraggingMovement.webm](https://github.com/dotnet/aspire/assets/90901519/cccd6ffb-ecea-4ed5-aec9-98a43cc5f770)

![Traces_Table_DraggingMovement](https://github.com/dotnet/aspire/assets/90901519/153c5a28-d56f-47c1-a181-cdf732f8df22)

![StructuredLogs_Table_DRaggingMovement](https://github.com/dotnet/aspire/assets/90901519/94672ba5-8789-4efd-9a53-6a7aebdda2cf)

**Comment by vnbaaij** [comment]
Hi @PoojaNamde ,

As I am not an a11y expert, can you elaborate on what 'single pointer functionalty' means exactly? Is it keyboard support? Or more?

**Comment by PoojaNamde** [comment]
@vnbaaij  Single pointer means pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures.

**Comment by vnbaaij** [comment]
@PoojaNamde ok, that part is clear. But that does not help to make it clearer to me how this should/could be used in resizing the columns.

Please describe in more detail what kind of solution needs to be implemented to resolve this a11y issue.

**Comment by adamint** [comment]
@vnbaaij assigning to you

**Comment by kvenkatrajan** [comment]
@adamint  - Vincent has a PR checked in. Please upgrade to new blazor fluent version to apply fix in 8.1 (main) - we will not be applying this to the 8.0 branch.

**Comment by vnbaaij** [comment]
Just a heads up, the package containing the pr (4.6.2) has not yet been released.

**Comment by kvenkatrajan** [comment]
Thanks @vnbaaij  - please let us know when its ready for integration. Maybe remove the blocked tag :)

**Comment by adamint** [comment]
@vnbaaij could you ping w

**Comment by adamint** [comment]
@vnbaaij is there any update with this?

**Comment by vnbaaij** [comment]
Package has been released (4.7.1) resize by keyboard functionality is working in there. See the examples in the demo site at https://www.fluentui-blazor.net/DataGrid#typicalusage

![grid-resize-keyboard 2](https://github.com/dotnet/aspire/assets/1761079/7b49dc5b-141d-45a1-bc05-c81669ee378e)

**Comment by adamint** [comment]
@PoojaNamde please re-test on main branch. This is resolved

**Comment by SwatiJo** [comment]
Achieving keyboard equivalence for a dragging operation does not meet this Success Criterion.there should be single pointer available. for ex -
[Fluent UI - Controls - React - DetailsList - Keyboard Column Reorder & Resize (microsoft.com)](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/keyboardcolumnreorderresize)

**Comment by vnbaaij** [comment]
@merriemcgaw would a per column option menu (like in the example) with plus/minus buttons that increase/decrease column width in discreet steps be fulfilling this requirement as well?
 I think a dialog where a user needs to fill in a width in pixels is a bit antiquated.

**Comment by merriemcgaw** [comment]
@vnbaaij I think that +/- buttons in a column options button would be a perfectly reasonable option. If you want to make sure it would be sufficient before implementation we can go to Office Hours.

**Comment by PoojaNamde** [comment]
@adamint 
Verified the bug in the environment
still able to repro it as we are not able to access the +/- option with keyboard, hence reactivating it.
Please find repro attachment for reference.

[Video 9-09 at 17.44 (1).webm](https://github.com/user-attachments/assets/8ee17741-c2e9-4866-870b-d95b3245bdc2)

**Comment by adamint** [comment]
#A11yFixed; #Closed
